Makefile

```diff
@@ -2,17 +2,19 @@ HOSTNAME=okta.com
 NAMESPACE=pam
 NAME=oktapam
 BINARY=terraform-provider-${NAME}
-# On verion changes, update tag-checks.yml
+# On version changes, update tag-checks.yml
 VERSION=0.2.1
 OS_ARCH=$(shell go env GOOS)_$(shell go env GOARCH)
 PLUGIN_DIR=~/.terraform.d/plugins
+INSTALL_TARGET=${PLUGIN_DIR}/${HOSTNAME}/${NAMESPACE}/${NAME}/${VERSION}/${OS_ARCH}
+TEMP_DIR=/tmp
 
 SET_VERSION=-ldflags "-X github.com/okta/terraform-provider-oktapam/oktapam/version.Version=${VERSION}"
 
 .DEFAULT_GOAL := install
 
-build:
-	go build -ldflags "-X github.com/okta/terraform-provider-oktapam/oktapam/version.Version=${VERSION}dev" -o ${BINARY}
+compile:
+	go build -o ${BINARY} ${SET_VERSION}dev
```

Comment:
Was getting a syntax error until i changed the order of the args.

Reply:
Build log showing the error: https://buildkite.com/scaleft/terraform-provider-oktapam-artifact/builds/55#01826a6f-1206-4307-9207-8a815d0b945c